garrysmod/lua/vgui/dvscrollbar.lua

```diff
@@ -68,6 +70,9 @@ function PANEL:Init()
 	self:SetSize( 15, 15 )
 	self:SetHideButtons( false )
 
+	-- Nicer default smooth
+	self:SetSmoothScroll( true )

```

Comment:
make it false by default

Reply:
What's the reason it's not smooth?

Reply:
It should be false by default:  your change should not affect other panels.

Reply:
And this is the reason why you will be stuck without chromium forever ^